bin/dfx-sns-deploy

```diff
@@ -23,22 +23,10 @@ export DFX_NETWORK
   echo "    dfx nns import --network-mapping $DFX_NETWORK=local"
   exit 1
 } >&2
```

Comment:
Stdout now has a bunch of other data in addition to the candid, which breaks parsing.  Thankfully the data we need is now saved to a JSON file, so we can get it from there instead.